### PR TITLE
pkp/pkp-lib#5178 Changed spans to h3 for article titles

### PR DESCRIPTION
**Pull request by thinkbulecount2**
From issue https://github.com/pkp/pkp-lib/issues/5178

### CODE REVIEW

File: templates/frontend/pages/search.tpl [hunk 1]
```diff
@@ -90,7 +90,7 @@
 	{* Search results, finally! *}
 	<div class="search_results">
 		{iterate from=results item=result}
-			{include file="frontend/objects/article_summary.tpl" article=$result.publishedSubmission journal=$result.journal showDatePublished=true hideGalleys=true}
+			{include file="frontend/objects/article_summary.tpl" heading="h3" article=$result.publishedSubmission journal=$result.journal showDatePublished=true hideGalleys=true}
```
Comment (NateWr) [inline templates/frontend/pages/search.tpl]: Is there a `<h2>` element on the `search.tpl` page? I did a quick scan and didn't see one.

Comment (NateWr) [reply templates/frontend/pages/search.tpl:93]: Did you have a chance to investigate this comment?

Comment (thinkbulecount2) [reply templates/frontend/pages/search.tpl:93]: Nope, didn't find any `<h2>`s!

Comment (thinkbulecount2) [reply templates/frontend/pages/search.tpl]: Fixed this in https://github.com/pkp/ojs/pull/2692/commits/34d5d51fb90e9b4164970c0c46c3193e2b897625

**Comment by NateWr** [comment]
This looks good everywhere except the journal homepage. When the current issue is shown on the homepage there are duplicate `<h2>` elements. 

headings

The `issueToc.tpl` template needs to be passed a heading level and be able to adjust it as necessary.

**Comment by thinkbulecount2** [comment]
Also found that full-issue galleys section has an `h2` -- is this something I should also change?

```			<h2 id="issueTocGalleyLabel">
				{translate key="issue.fullIssue"}
			</h2>
```

**Comment by thinkbulecount2** [comment]
In `indexJournal.tpl` is it still appropriate to say

```
{if !$heading}
	{assign var="heading" value="h3"}
{/if}
```

when there are clear headings or `<h2>`s throughout? Should I rename `$heading` to something more specific, e.g. `issueHeading`?

**Comment by NateWr** [comment]
> Also found that full-issue galleys section has an h2 -- is this something I should also change with a $heading variable?

Good spot, yes!

> In indexJournal.tpl

Because `indexJourna.tpl` is a one-off, I think you don't need a `$heading` variable. You can explicitly pass in the heading level that you want:

```
{include ... heading="h2"} // or h3 or whatever
```

You only need the variable in the templates that get loaded in by another template, like `article_summary.tpl` or `issue_toc.tpl`

**Comment by thinkbulecount2** [comment]
Check now, @NateWr?

**Comment by NateWr** [comment]
Replaced by https://github.com/pkp/ojs/pull/2751

**Pull request closed without merging**